frontend/packages/dev-console/src/components/helm/HelmActionForm.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { FormikProps, FormikValues, useFormikContext } from 'formik';
+import { safeDump } from 'js-yaml';
+import { TextInputTypes, Grid, GridItem } from '@patternfly/react-core';
+import { coFetchJSON } from '@console/internal/co-fetch';
+import { InputField, FormFooter, FlexForm, YAMLEditorField, DropdownField } from '@console/shared';
+import FormSection from '../import/section/FormSection';
+import { HelmActionType } from './helm-types';
+
+interface HelmActionFormProps {
+  chartHasValues: boolean;
+  chartVersions?: {};
+  activeChartVersion?: string;
+  onChartVersionChange?: (value: string) => string;
+  submitLabel: string;
+}
+
+const HelmActionForm: React.FC<FormikProps<FormikValues> & HelmActionFormProps> = ({
+  chartHasValues,
+  errors,
+  handleSubmit,
+  handleReset,
+  status,
+  isSubmitting,
+  chartVersions,
+  activeChartVersion,
+  onChartVersionChange,
+  submitLabel,
+  dirty,
+}) => {
+  const { setFieldValue } = useFormikContext<FormikValues>();
+  return (
+    <FlexForm onSubmit={handleSubmit}>
+      <FormSection fullWidth>
+        <Grid gutter={'md'}>
+          <GridItem span={submitLabel === HelmActionType.Install ? 12 : 6}>
+            <InputField
+              type={TextInputTypes.text}
+              name="helmReleaseName"
+              label="Release Name"
+              helpText="A unique name for the Helm Chart release."
+              required
+              isDisabled={!_.isEmpty(activeChartVersion)}
+            />
+          </GridItem>
+          {activeChartVersion && (
+            <GridItem span={6}>
+              <DropdownField
+                name="chartVersion"
+                label="Chart Version"
+                items={chartVersions}
+                fullWidth
+                helpText={'Select the version to upgrade to.'}
+                required
+                disabled={_.isEmpty(chartVersions)}
+                title={activeChartVersion}
+                onChange={(value) => {
+                  const url = onChartVersionChange(value);
```

Comment:
Why not just use `getChartURL` util function here?

Reply:
same as https://github.com/openshift/console/pull/4785#discussion_r399575853

Reply:
Made the requested change